sale_order_import/tests/test_sale_order.csv

```diff
@@ -0,0 +1,2 @@
+Group,Line Product,Line Description,Line Unit Price,Line Qty,Line Tax,Customer,Pricelist,Warehouse,Notes,Carrier,Team,Customer Phone/Mobile
+,ABC,Product A,200,2,,Test Partner,Public Pricelist,YourCompany,Test Note,Free delivery charges,America,+1234567890
```

Comment:
For safe, I think we need to have 3 lines, one order with 1 order line and one order with 2 order lines